app.py

```diff
@@ -52,33 +51,36 @@ def run_app(self):
         (current_in, current_out) = self.get_current_frame_range(self.engine.name)
 
         if new_in is None or new_out is None:
-            message =  "Shotgun has not yet been populated with \n"
-            message += "in and out frame data for this Shot."
+            message = ("Shotgun has not yet been populated with \n"
+                       "in and out frame data for this Shot.")
```

Comment:
In this case @gst , lets not pollute existing code not related to the frame range. Thanks!